### DISCUSSION TITLE
What Stuff i have to know before i contribute ??

### DISCUSSION THREAD

**Discussion by Prathamix77** [facebook/docusaurus#9501] [Q&A] [answered]
I wanna know the tech stack and also the resources , I need to learn before helping up the project
PS : I really wanna contribute into this project

THANK YOU :)###

**Comment by slorber** [comment, marked as answer]
You can just read our doc and help qualify issues, write docs or provide support, no skills required

Otherwise React typescript node MDX webpack...
